packages/zambdas/src/ehr/get-lab-orders/helpers.ts

```diff
@@ -124,17 +130,23 @@ export const parseOrderDetails = <SearchBy extends LabOrdersSearchBy>({
   const appointmentId = parseAppointmentId(serviceRequest, encounters);
   const appointment = appointments.find((a) => a.id === appointmentId);
   const { testItem, fillerLab } = parseLabInfo(serviceRequest);
+  const location = locations.find((location) => {
+    const locationRef = `Location/${location.id}`;
+    return serviceRequest.locationReference?.find((srLocationRef) => srLocationRef.reference === locationRef);
+  });
+
+  const timezone = location ? getTimezone(location) : undefined;
```

Comment:
```  
const location = locations.find((location) => {
    const locationRef = `Location/${location.id}`;
    return serviceRequest.locationReference?.find((srLocationRef) => srLocationRef.reference === locationRef);
  });

  onst timezone = location ? getTimezone(location) : undefined;
```

Let's extract this into a helper. The code is easier to read when a function uses the same level of abstraction:

```
const timezone = parseTimezone(serviceRequest, locations);
```